src/Uno.SourceGeneration.Engine.Shared/Workspace/ProjectFileInfo.cs

```diff
@@ -44,6 +46,19 @@ internal sealed class ProjectFileInfo
         /// </summary>
         public string OutputRefFilePath { get; }
 
+        // /// <summary>
+        // /// The default namespace of the project ("" if not defined, which means global namespace),
+        // /// or null if it is unknown or not applicable. 
+        // /// </summary>
+        // /// <remarks>
+        // /// Right now VB doesn't have the concept of "default namespace". But we conjure one in workspace 
+        // /// by assigning the value of the project's root namespace to it. So various feature can choose to 
+        // /// use it for their own purpose.
+        // /// In the future, we might consider officially exposing "default namespace" for VB project 
+        // /// (e.g. through a "defaultnamespace" msbuild property)
+        // /// </remarks>
+        // public string DefaultNamespace { get; }
```

Comment:
I synced the code from Roslyn and commented out things we're not using.

Reply:
for large chunk, you can wrap them in `#if false`

Reply:
for large chunk, you can wrap them in `#if false`